tests/AppBundle/Spreadsheet/DeliverySpreadsheetParserTest.php

```diff
@@ -56,18 +62,17 @@ protected function createParser(): AbstractSpreadsheetParser
 
         $this->packageRepository = $this->prophesize(ObjectRepository::class);
 
-        $this->entityManager
-            ->getRepository(Package::class)
-            ->willReturn($this->packageRepository->reveal());
+        $this->tourRepository = self::getContainer()->get(TourRepository::class);
```

Comment:
maybe a test fails, because it should be a mock?

Reply:
mmm I dont think so
I ve had weird doctrine issues when working on such parts of the code
I think it is mostly due that there are so much listeners crossed that sometimes doctrine lose track of what is where